exercises/concept/squeaky-clean/tests/Tests.elm

```diff
@@ -0,0 +1,112 @@
+module Tests exposing (tests)
+
+import Expect
+import SqueakyClean exposing (..)
+import Test exposing (Test, describe, skip, test)
+
+
+tests : Test
+tests =
+    describe "Squeaky Clean"
+        [ describe "Clean1" [ testTask1 clean1 ]
+        , skip <|
```

Comment:
We don't need skips on concept exercises because they are designed to be online only.

Reply:
Yes, I think you might be right, there is code that removes it, but still, there is no point in having something useless in the code. For practice exercises it makes sense because the students are able to download them and solve them locally one test at a time.
The docs mention skips for practice exercises, but they don't for concept exercises, so it's in there :)

Reply:
Weeeell... Technically you can 😇 
But they are very much created for the online editor, with showing the concepts first, breaking down the exercises in separate parts following a story, the hints, then showing you the full concept when you succeed...

Reply:
Right, got it! And actually all the other concept exercises don't have skips either, so good to be consistent with them as well.

Reply:
Got it. I am removing the skips and structuring the tests similar to other concept exercises.